aws_lambda_powertools/shared/functions.py

```diff
@@ -275,13 +275,11 @@ def abs_lambda_path(relative_path: str = "") -> str:
         If the path is empty, it will return the current working directory.
     """
     # Retrieve the LAMBDA_TASK_ROOT environment variable or default to an empty string
-    current_working_directory = os.environ.get("LAMBDA_TASK_ROOT", "")
+    current_working_directory = os.environ.get("LAMBDA_TASK_ROOT", "") or str(Path.cwd())
```

Comment:
```suggestion
    current_working_directory = os.environ.get("LAMBDA_TASK_ROOT", str(Path.cwd()))
```

Do we really need to "or" ?

Reply:
Good catch!!!

Reply:
Ops sorry! I think we need to keep the `or` because `LAMBDA_TASK_ROOT` may exist, but it may be an empty string. Keeping as is.